cmd/lima-guestagent/daemon_linux.go

```diff
@@ -26,16 +27,24 @@ func newDaemonCommand() *cobra.Command {
 	return daemonCommand
 }
 
+var (
+	vSockPort = 0
+
+	qemuFile = "/dev/virtio-ports/lima.guest_agent.0"
```

Comment:
Could you document these virtio ports in https://lima-vm.io/docs/dev/internals/ ?

Reply:
Also, "qemuFile" sounds confusing.
Maybe this should be called `virtioPort` ?

Reply:
```suggestion
	virtioPort = "/dev/virtio-ports/io.lima-vm.guest_agent.0"
```